sdk/python/cmd/pulumi-language-python/main.go

```diff
@@ -1100,6 +1101,7 @@ func validateVersion(ctx context.Context, options toolchain.PythonOptions) {
 		return
 	}
 	version := strings.TrimSpace(strings.TrimPrefix(string(out), "Python "))
+	version = removeReleaseCandidateSuffix(version)
```

Comment:
This is not strictly necessary anymore, but will come handy if we want to try out the 3.14 release candidates next time around :)